File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        String[] arr = s.split(" ");

        int min = Integer.MAX_VALUE;
        int max = Integer.MIN_VALUE;

        for (String num : arr) {
            int n = Integer.parseInt(num);
            min = Math.min(min, n);
            max = Math.max(max, n);
        }

        String result = min + " " + max;
        return result;
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <string>
#include <sstream>
#include <algorithm>

using namespace std;

string solution(string s) {
    stringstream ss(s);
    int num;
    
    ss >> num;
    int min_num = num;
    int max_num = num;
    
    while (ss >> num) {
        min_num = min(min_num, num);
        max_num = max(max_num, num);
    }
    
    return to_string(min_num) + " " + to_string(max_num);
}